src/index.css

```diff
@@ -86,6 +86,8 @@ a {
   .header-search {
     flex-grow: 1;
     max-height: 48px;
+    margin-right: 0 !important;
+    margin-left: 0 !important;
```

Comment:
If we didn't update any styles from library components, we should not use `!important.` It's an antipattern